lib/ddtrace/contrib/active_record/patcher.rb

```diff
@@ -0,0 +1,83 @@
+module Datadog
+  module Contrib
+    module ActiveRecord
+      # Patcher enables patching of 'active_record' module.
+      # This is used in monkey.rb to manually apply patches
+      module Patcher
+        @patched = false
+
+        module_function
+
+        # patched? tells whether patch has been successfully applied
+        def patched?
+          @patched
+        end
+
+        def patch
+          if !@patched && defined?(::ActiveRecord)
+            begin
+              require 'ddtrace/contrib/rails/utils'
+              require 'ddtrace/ext/sql'
+
+              patch_active_record()
+
```

Comment:
Reading recent Redis bug-fix concerning service type, I think we might need something that looks like this https://github.com/DataDog/dd-trace-rb/blob/98d96aeab9cfb5ae7a1d12a520c550be22828a63/lib/ddtrace/contrib/redis/patcher.rb#L65 or this https://github.com/DataDog/dd-trace-rb/blob/master/lib/ddtrace/contrib/rails/framework.rb#L75 here (`set_service_info` must be explicitly called, else service type defaults to custom, the consequence being the wrong icon displayed in UI listings)